cryptus-app/components/market_overview/net_worth/net_worth.module.scss

```diff
@@ -0,0 +1,48 @@
+@use "/styles/variables" as v;
+
+.container {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: v.$bg-color;
+}
+.container_row {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: v.$bg-color;
+}
+
+.value {
+  font-family: "Arial Black", Gadget, sans-serif;
+  font-size: 36px;
+  color: #000000;
+  font-style: normal;
+  color: black;
+}
+
+.delta {
+  font-family: "Arial Black", Gadget, sans-serif;
+  font-size: 18px;
+  color: #000000;
+  font-style: normal;
+  color: black;
+}
+
+.arrowdown {
+  margin: 6px;
+  border: solid red;
+  transform: rotate(45deg);
+  border-width: 0 3px 3px 0;
+  display: inline-block;
+  padding: 3px;
+}
+
+.arrowup {
+  border: solid green;
+  transform: rotate(-135deg);
+  -webkit-transform: rotate(-135deg);
```

Comment:
remove the unused one

Reply:
We need this CSS for when we will have a green arrow, will be used later